packages/cli/src/index.ts

```diff
@@ -94,11 +94,13 @@ cli
         compilation: {
           mode: options.mode,
           watch: options.watch,
-          output: options.outDir
-            ? {
-                path: options.outDir
-              }
-            : undefined,
+          output:
+            options.output.outDir || options.output.format
```

Comment:
Options as a cli command line object is not a property of a compilation object, so there is no output object